docs/contribute/index.md

```diff
@@ -37,3 +47,86 @@ To contribute to earlier versions of the Elastic Stack, you must work with our [
+
+#### What is staying in the Elasticsearch repo?
+
+* Reference content (anything that is or could be auto-generated): Settings, syntax references, functions lists…  
+* Contribution guides and extending elasticsearch (clients, etc)  
+* Release notes
+
+### Cloud
+
+#### What has moved to elastic/docs-content?
+
+Public-facing narrative and conceptual docs have moved. Most can now be found under the following directories in the new docs:
+
+* **deploy-manage**: deploying cloud
+* **cloud-account**: managing your cloud organization  
+* **troubleshooting**: troubleshooting pages
+
+#### **What is staying in the Cloud repo?**
+
+* Reference content (= anything that is or could be auto-generated): Settings, syntax references, functions lists…  
+* Release notes
+
+### Machine learning
+
+#### What has moved to elastic/docs-content?
+
+Public-facing narrative and conceptual docs have moved. Most can now be found under the following directories in the new docs:
+
+* **deploy-manage**: deploying cloud, managing your cloud organization  
+* **troubleshooting**: troubleshooting pages
+
+#### What is staying in the Elasticsearch repo?
+
+* Reference content, anything that is or could be auto-generated): Settings, syntax references, functions lists…  
+* Release notes
+
+## IA Sections to product areas
+
+The following section shows a mapping of the different areas in the docs IA to the product areas and topics.
+
+| **Area** | **Description** | **Sources (Area / Topic)** |
+| ----- | ----- | ----- |
+| **Get started** | Content in this section focuses on learning about Elasticsearch and the stack, learning about how it can be deployed, and basic environment setup for initial exploration. | Overview of Elastic and various topics |
+| **Solutions and use cases** | Content in this section focuses on the core solutions themselves and their user cases. | **Search:** core search solution content, playground, hybrid search <br/>**Observability:** core observability content <br/>**Security:** core security content  |
+| **Manage data** | Content in this section focuses on learning about Elastic data store primitives, evaluating and implementing ingestion and data enrichment technologies, managing the lifecycle of your data and migrating data between clusters. | **Elasticsearch:** data store, migration, data lifecycle <br/>**Ingest:** ingesting data into Elasticsearch <br/>**Kibana**: Discover, Dashboards, Visualizations, Reporting, Alerting, dev tools. |
+| **Explore and analyze** | Content in this section focusing on querying, visualizing, and exploring data. Additionally it focusing on leveraging machine learning for data analysis and defining and responding to alerts |  **Elasticsearch:**  query languages, scripting, machine learning (NLP)  <br/>**Machine Learning:** Anomaly detection, data frame analytics, NLP <br/>**Kibana**: Discover, Dashboards, Visualizations, Reporting, Alerting, dev tools. <br/>**ResponseOps:** Alerts and Cases |
```

Comment:
```suggestion
| **Explore and analyze** | Content in this section focuses on querying, visualizing, and exploring data. Additionally it focusing on leveraging machine learning for data analysis and defining and responding to alerts |  **Elasticsearch:**  query languages, scripting, machine learning (NLP)  <br/>**Machine Learning:** Anomaly detection, data frame analytics, NLP <br/>**Kibana**: Discover, Dashboards, Visualizations, Reporting, Alerting, dev tools. <br/>**ResponseOps:** Alerts and Cases |
```

I'd actually lean towards deleting the repetitive "Content in this section focuses on" in all rows just to keep concise